mopidy_musicbox_webclient/static/js/gui.js

```diff
@@ -508,11 +509,13 @@ $(document).ready(function(event) {
 
     //navigation temporary, rewrite this!
     $('#songinfo').click(
-	function()
-    	 {return switchContent('nowPlaying')}   );
+	function() {
+        return switchContent('nowPlaying');
+    });
```

Comment:
I'm trying out Sublime Text for the first time and didn't notice that it doesn't convert tabs to spaces by default.

I'll fix this and a few other spacing issues and just push it directly to develop.